Lib/test/test_socketserver.py

```diff
@@ -55,7 +55,7 @@ class ForkingUnixDatagramServer(socketserver.ForkingMixIn,
                                     socketserver.UnixDatagramServer):
         pass
 
-@unittest.expectedFailureIfWindows("Fork is for Unix based systems only")
+@test.support.requires_fork()
```

Comment:
This might need a `TODO: RUSTPYTHON` comment:

```suggestion
@test.support.requires_fork() # TODO: RUSTPYTHON, os.fork is currently only supported on Unix-based systems
```